{"name": "导入老师", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'files/老师模版.xlsx'", "trace": "self = <testcases.test_user_manage.TestUserManageApi object at 0x0000020ACAC5F310>\n\n    @allure.story(\"导入老师测试用例\")\n    @allure.title(\"导入老师\")\n    @allure.severity(\"normal\")\n    @allure.description(\"导入老师\")\n    def test_import_teacher(self):\n>       self.user_manage.import_teacher()\n\ntest_user_manage.py:96: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <api.UserManageApi.UserManageApi object at 0x0000020ACACD45B0>\n\n    def import_teacher(self):\n        \"\"\"导入老师\"\"\"\n        path = \"backend/user/import-teacher\"\n        files = {\n            \"upload_file\": (\n                \"老师模版.xlsx\",\n>               open(r\"files/老师模版.xlsx\", \"rb\"),\n                \"application/octet-stream\"\n            )\n        }\nE       FileNotFoundError: [Errno 2] No such file or directory: 'files/老师模版.xlsx'\n\n..\\api\\UserManageApi.py:67: FileNotFoundError"}, "description": "导入老师", "start": 1723024204089, "stop": 1723024204089, "uuid": "961be284-55f1-4384-b13f-8c9a0ee5ed19", "historyId": "0e2af3315507c8fcb4346c6e7f18fc30", "testCaseId": "0e2af3315507c8fcb4346c6e7f18fc30", "fullName": "testcases.test_user_manage.TestUserManageApi#test_import_teacher", "labels": [{"name": "feature", "value": "用户管理模块"}, {"name": "story", "value": "导入老师测试用例"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_manage"}, {"name": "subSuite", "value": "TestUserManageApi"}, {"name": "host", "value": "常胜峰"}, {"name": "thread", "value": "27540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_manage"}]}